rust/worker/src/compactor/compaction_manager.rs

```diff
@@ -95,7 +96,7 @@ impl CompactionManager {
         compaction_job: &CompactionJob,
     ) -> Result<CompactionResponse, Box<dyn ChromaError>> {
         let dispatcher = match self.dispatcher {
-            Some(ref dispatcher) => dispatcher,
+            Some(ref dispatcher) => dispatcher.clone(),
```

Comment:
do we need to clone here?

Reply:
yes, because CompactOrchestrator takes ownership